mock/py/mockbuild/scm.py

```diff
@@ -115,14 +115,14 @@ def adjust_git_timestamps(self):
         self.log.debug("Adjusting timestamps in %s", self.src_dir)
         os.chdir(self.src_dir)
         proc = subprocess.Popen(['git', 'ls-files', '-z'], shell=False, stdout=subprocess.PIPE)
-        for f in proc.communicate()[0].split('\0')[:-1]:
+        for f in proc.communicate()[0].split(b'\0')[:-1]:
```

Comment:
Would you mind using `Popen(..., universal_newlines=True)` instead?

Reply:
... thank you for the patch!

Reply:
Okay, it's updated, thanks.